VisualPIC/DataHandling/unitConverters.py

```diff
@@ -317,8 +318,14 @@ def SetSimulationParameters(self, params):
         super().SetSimulationParameters(params)
         self._SetNormalizationFactor(None)
 
-    def ConvertToISUnits(self, dataElementName, data):
-        return data
+    def ConvertToISUnits(self, dataElement, data):
```

Comment:
Is `IS` a typo and means `SI`?

Reply:
Yep, typo. There are probably a lot more in the code, but I'll fix them eventually in a different branch.